bin/ch-test

```diff
@@ -784,7 +822,20 @@ else
 fi
 printf 'contributor:  %s\n\n' "$ch_contributor_note"
 
+if [[ $phase = one-file ]]; then
+    if [[ ! -f $one_file ]]; then
+        fatal "not a file: $one_file"
+    fi
+    one_file=$(readlink -f "$one_file")  # make absolute b/c we cd later
+    if [[ $one_file = */test.bats && -z $one_file_tag ]]; then
+        fatal 'must specify --file-tag for --file=*/test.bats'
```

Comment:
I believe this if condition and error message needs an update. Other than that, looks good to me.

Reply:
Good catch; thank you.